client/settings/patch-starcraft.js

```diff
@@ -89,11 +95,23 @@ async function patchFile(dirPath, outPath, filename, validHashes) {
   }
 }
 
+async function copyLocalDll(dirPath, downgradePath) {
+  const matches = await globAsync(`${dirPath}/**/local.dll`)
```

Comment:
Apply the same advice in the check function to here as far as finding the DLL.

This also potentially causes problems if they have multiple locales installed (and we pick the local.dll for a locale they don't want). I think that's okay for now, especially if we plan on letting people select locales down the road and replace local.dll, but something to be aware of.